index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+require('dotenv').config();
+
+const mongoose = require('mongoose');
+
+const MONGODB_URI = process.env.MONGODB_URI;
+const mongoOptions = {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+    useCreateIndex: true
+}
+mongoose.connect(MONGODB_URI, mongoOptions);
+
+require('./src/app.js').start(process.env.PORT);
```

Comment:
Looks like your .gitignore is not preventing your package-lock.json from not being pushed up